core/src/test/java/ch/cyberduck/core/serializer/impl/dd/ProfilePlistReaderTest.java

```diff
@@ -270,4 +271,29 @@ public boolean isEnabled() {
         assertEquals("STANDARD OTHER", properties.get("s3.storage.class.options"));
         assertEquals(Arrays.asList("STANDARD", "OTHER"), PreferencesReader.toList(properties.get("s3.storage.class.options")));
     }
+
+    @Test
+    public void testSerializationFull() throws Exception {
+        final ProfilePlistReader reader = new ProfilePlistReader(new ProtocolFactory(Collections.singleton(new TestProtocol() {
+            @Override
+            public Type getType() {
+                return Type.s3;
+            }
+
+            @Override
+            public boolean isEnabled() {
+                return false;
+            }
+
+            @Override
+            public String getIdentifier() {
+                return "MyId";
+            }
+        })));
+        final Profile profile = reader.read(
+                new Local("src/test/resources/full.cyberduckprofile")
```

Comment:
Use `Paths.get(resource.toURI()).toFile().getAbsolutePath()` in order to not depend on current working directory.